### PR TITLE
move locate to the cloud

### PR DESCRIPTION
**Pull request by steveoh**
firebase hosting via github actions

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+name: firebase deploy
+
+on:
+  push:
+    paths:
+      - "_src/**"
```
Comment (stdavis) [inline .github/workflows/nodejs.yml:6]:
```suggestion
      - "lib/**"
```

This app was started before we standardized on `_src`

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      - ".github/workflows/nodejs.yml"
+      - "package.json"
```
Comment (stdavis) [inline .github/workflows/nodejs.yml:8]: Should we add `bower.json` to this? I didn't think about this when reviewing raster but that's probably more important than package.json.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 # Broadband Economic Development
 
-[![Build Status](https://travis-ci.org/agrc/bb-econ.svg?branch=master)](https://travis-ci.org/agrc/bb-econ)
+[![build status](https://github.com/agrc/locate/actions/workflows/nodejs.yml/badge.svg)](https://github.com/agrc/locate/actions/workflows/nodejs.yml)
+
 [![Sauce Test Status](https://saucelabs.com/browser-matrix/agrc-bb-econ.svg)](https://saucelabs.com/u/agrc-bb-econ)
 
-Production URL: [locate.utah.gov](https://locate.utah.gov)  
+Production URL: [locate.utah.gov](https://locate.utah.gov)
 Staging URL: [test.mapserv.utah.gov/bb-econ](http://test.mapserv.utah.gov/bbecon)
```
Comment (stdavis) [inline README.md:8]:
```suggestion
Staging URL: [locate.dev.utah.gov](https://locate.dev.utah.gov)
```

**Review by stdavis — CHANGES_REQUESTED** [review]
This should run a test job as well, no? Or are they broken?

Also, looks like the npm lock file needs to be updated.

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (steveoh) [reply .github/workflows/nodejs.yml:8]: we certainly can.

**Review by stdavis — APPROVED** [review]
Assuming that the app isn't broken, these changes look good to me.

**Comment by steveoh** [comment]
> Assuming that the app isn't broken, these changes look good to me.

Can you check? Some things seem fouled up to me but I'm not super familiar.

**Comment by nathankota** [comment]
RE: 

> Assuming the app isn't broken...

Summary report not working on dev site.

**Comment by stdavis** [comment]
> Some things seem fouled up to me but I'm not super familiar.

I've got the report fixed. What else is there that's broken? I can't find anything. The map service seems to be drawing quickly to me.

**Pull request merged**